theano/gpuarray/c_code/dimshuffle.c

```diff
@@ -0,0 +1 @@
+#section support_code
```

Comment:
In fact, some important attributes like `self.suffle`, `self.drop`, `self.new_order` are created and checked  into DimShuffle `__init__()`, so this method is necessary for GpuDimShuffle too.